packages/contact-center/cc-components/package.json

```diff
@@ -20,7 +20,7 @@
     "build": "yarn run -T tsc",
     "build:src": "yarn run clean:dist && webpack",
     "build:watch": "webpack --watch",
-    "test:unit": "jest"
+    "test:unit:broken": "jest"
```

Comment:
We've marked it broken temporarily. We are working with Christoph's team to fix it and there's a Bug Kanban board sticky for it.